integrated_channels/moodle/exporters/content_metadata.py

```diff
@@ -112,3 +98,23 @@ def transform_end(self, content_metadata_item):
         if end_date:
             return int(parse(end_date).timestamp())
         return None
+
+    def transform_announcement(self, content_metadata_item):
+        """
+        Formats a post in a course's Announcement forum with
+        edX's course summary and enrollment link
+        """
+        full_description = content_metadata_item.get('full_description') or None
+        short_description = content_metadata_item.get('short_description') or None
+
+        if full_description and len(full_description) <= (self.LONG_STRING_LIMIT-400):
```

Comment:
interesting, perhaps worth a comment here on why the -400 ?

Reply:
I'm actually just going to bake that into the long_string_limit var and explain the arbitrary buffer going on there.